proto/streamlit/proto/Element.proto

```diff
@@ -66,6 +67,7 @@ message Element {
     Balloons balloons = 12;
     BokehChart bokeh_chart = 17;
     Button button = 19;
+    DownloadButton download_button = 42;
```

Comment:
Since we didn't coordinate on this, I think I put Metric as 42. Should I change my PR or just change it when we merge?